addons/portal/data/mail_template_data.xml

```diff
@@ -5,10 +5,14 @@
         <record id="mail_template_data_portal_welcome" model="mail.template">
             <field name="name">Portal: User Invite</field>
             <field name="model_id" ref="portal.model_portal_wizard_user"/>
-            <field name="subject">Your account at {{ object.user_id.company_id.name }}</field>
-            <field name="email_to">{{ object.user_id.email_formatted }}</field>
+            <field name="subject">Your account at {{ object.user_id.company_id.name or object.partner_id.company_id.name or ''}}</field>
```

Comment:
False should format to an empty string, '' fallback is more of a js template issue usually ^^
```suggestion
            <field name="subject">Your account at {{ object.user_id.company_id.name or object.partner_id.company_id.name }}</field>
```